### PR TITLE
added publisher read only mode docs

### PR DESCRIPTION
**Pull request by hasuniea**
https://docs.google.com/document/d/1VNlv2HEB7EN_kk_57GmA65KtqO5vTu2eYjjS83VFf9E/edit?usp=sharing

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Comment by 1akshitha** [comment]
@Mariangela can we please get this merged ASAP. This is a blocker for the M3 release which is scheduled for this week.

File: en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Publisher Portal in Read-only Mode
+A user who has the view/read only permissions can only view the API, API Product and Service details in the publisher
```
Comment (Mariangela) [inline en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md:2]:
```suggestion
A user who has the view/read only permissions can only view the API, API Product and Service details in the Publisher
```

File: en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+portal and also he/she can review all the analytics related details.
+
+#### Read Only User:
```
Comment (Mariangela) [inline en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md:5]:
```suggestion
#### Read Only User
```

File: en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+The user has only the apim:api_view permission.
```
Comment (Mariangela) [inline en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md:6]:
```suggestion
The user has only the `apim:api_view` permission.
```

File: en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+Let's create a read only user and experience the publisher portal in read only mode.
```
Comment (Mariangela) [inline en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md:8]:
```suggestion
Let's create a read only user and experience the Publisher portal in read only mode.
```

File: en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+Steps to create a Read only user:
```
Comment (Mariangela) [inline en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md:10]:
```suggestion
Follow the instructions below to create a Read only user:
```

File: en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+1. Create role ex: read_only in admin console
```
Comment (Mariangela) [inline en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md:11]:
```suggestion
1. Create role (e.g., `read_only`) in admin console.
```

File: en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+2. Create a user and assign the read_only role to the user.
+3. Assign the login permission to login to the publisher portal via the admin console.
```
Comment (Mariangela) [inline en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md:13]:
```suggestion
3. Assign the login permission to login to the Publisher portal via the admin console.
```

File: en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+4. Login to the admin portal and navigate to scope management under the Settings subheading.
```
Comment (Mariangela) [inline en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md:14]:
```suggestion
4. Log in to the Admin Portal and navigate to **scope management** which is under **Settings**.
```

File: en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+5. Click on the add scope mappings.
```
Comment (Mariangela) [inline en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md:15]:
```suggestion
5. Click **Add Scope Mappings**.
```

File: en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+   Under the Role name please give the Role as read_only(the role you assigned to the read only user).
```
Comment (Mariangela) [inline en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md:16]:
```suggestion
   Enter the Role name as `read_only` (This is the role you assigned to the read only user).
```

File: en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+6. Click on the Next button.
+7. Please select the Custom scope assignments under the select permissions.
+   Check apim:api_view permission under the publisher.
```
Comment (Mariangela) [inline en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md:19]:
```suggestion
   Check the `apim:api_view` permission under the Publisher.
```

File: en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+
+   ![read-only-user-creation]({{base_path}}/assets/img/learn/api-security/read-only-user-creation.png)
+
+8. Save the changes.
+
+If a read-only user (jane) is logged in to the publisher,she can see the publisher portal as in below.
+![read-only-user-publisher-view]({{base_path}}/assets/img/learn/api-security/read-only-api-view.png)
+
+ex: API detail view
```
Comment (Mariangela) [inline en/docs/design/api-security/authorization/publisher-portal-in-read-only-mode.md:28]:
```suggestion
Example: API detail view
```

File: en/docs/get-started/about-this-release.md [hunk 13]
```diff
@@ -38,6 +38,11 @@ API Manager 4.1.0-M2</td>
 
 	 Authentication for Prototype APIs is available by default so that the client apps can also be created using authentication to test out the APIs. If needed you can [disable authentication for Prototype APIs]({{base_path}}/design/prototype-api/create-a-prototype-api/)) at the resource level or API level.
 
+- **Publisher Read Only Mode (M3)** 
+    
+     A user who has view/read only permissions can only view the API/Services details in the publisher portal.He/She should not be able to edit/create/publish APIs/Services.
```
Comment (Mariangela) [inline en/docs/get-started/about-this-release.md:43]:
```suggestion
     A user who has view/read only permissions can only view the API/Services details in the Publisher portal. Such users should not be able to edit/create/publish APIs/Services.
```

**Review by Mariangela — CHANGES_REQUESTED** [review]
@hasuniea , please note my comment and see the suggestions I have added please.

File: en/mkdocs.yml [hunk 14]
```diff
@@ -1830,6 +1830,7 @@ plugins:
               'learn/api-security/authorization/api-authorization.md': 'https://apim.docs.wso2.com/en/4.0.0/design/api-security/authorization/api-authorization/'
               'learn/api-security/oauth2/oauth2-scopes/fine-grained-access-control-with-oauth-scopes.md': 'https://apim.docs.wso2.com/en/4.0.0/design/api-security/oauth2/oauth2-scopes/fine-grained-access-control-with-oauth-scopes/'
               'learn/api-security/authorization/role-based-access-control-using-xacml.md': 'https://apim.docs.wso2.com/en/4.0.0/design/api-security/authorization/role-based-access-control-using-xacml/'
+              'learn/api-security/authorization/publisher-portal-in-read-only-mode.md': 'https://apim.docs.wso2.com/en/4.0.0/design/api-security/authorization/publisher-portal-in-read-only-mode/'
```
Comment (Mariangela) [inline en/mkdocs.yml:1834]:
```suggestion
```

**Review by Mariangela — APPROVED** [review]

**Pull request merged**